### DISCUSSION TITLE
[help] `tar_make()` doesn't show any color in RStudio console

### DISCUSSION THREAD

**Discussion by Rekyt** [ropensci/targets#1271] [Help]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Hi there,
I'm having a very similar issue to https://github.com/ropensci/targets/discussions/302
Namely that `tar_make()` doesn't show any output in colors in the RStudio console.

I thought it was an issue with my platform or my RStudio config but it seems to be targets only issue as I managed to get syntax coloring with `usethis` (see the screenshot below):

![image](https://github.com/ropensci/targets/assets/5593751/fdf1beeb-4960-4179-9af8-bc6ab6f9679e)

I've checked the environment variables or the `cli` options but nothing has been set so `targets` should display the workflow in colors.
I've also try adding `options(crayon.enabled = TRUE)` at the beginning of the `_targets.R` file, and it doesn't change the result.

What would cause such a difference in behavior between targets and other packages?

Thanks!

**Comment by wlandau** [comment]
The red text comes from how RStudio and `callr` handle colors, c.f. https://github.com/rstudio/rstudio/issues/2574 and https://github.com/r-lib/cli/issues/467. So it is outside the control of `targets` and would need to be addressed in one or both of those other tools.
